build/rollup-config.js

```diff
@@ -23,15 +24,33 @@ const banner = `/* @preserve
  */
 `;
 
+const outro = `var oldL = window.L;
+function noConflict() {
```

Comment:
This has to be changed to `exports.noConflict = function()` too.

Reply:
If not, the UMD bundle for the tests will work, but the one for release won't. 😐